### ISSUE TITLE
USWDS-Site - Documentation: Improve clarity in sample layout settings

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds-site#2611]
_From a ZenDesk question._

>To turn on or off breakpoints as mentioned on this page
https://designsystem.digital.gov/utilities/layout-grid/#sass-mixins
, what is the name/location of the scss file that I should modify?

## Problem

It's not clear where these settings should be added in the code examples.
![image](https://github.com/uswds/uswds-site/assets/3385219/8af24065-054f-4e1e-acc9-1f621e56116a)

## Expected

A file name or more accurate example of how they are used should be added.

A possible solution:

```diff

+ /* _uswds-theme.scss */

// Values are set as units tokens.

- $theme-column-gap-sm:               2px;
-  $theme-column-gap-md:               2;
-  $theme-column-gap-lg:               3;
-  $theme-column-gap-mobile:           2;
-  $theme-column-gap-desktop:          4;
-  $theme-grid-container-max-width:    'desktop';
-  $theme-site-margins-breakpoint:     'desktop';
-  $theme-site-margins-width:          4;
-  $theme-site-margins-mobile-width:   2;

+@use "uswds-core" with (
+ $theme-column-gap-sm:               2px;
+  $theme-column-gap-md:               2;
+  $theme-column-gap-lg:               3;
+  $theme-column-gap-mobile:           2;
+  $theme-column-gap-desktop:          4;
+  $theme-grid-container-max-width:    'desktop';
+  $theme-site-margins-breakpoint:     'desktop';
+  $theme-site-margins-width:          4;
+  $theme-site-margins-mobile-width:   2;
+ )
```

Source: [_utilities/layout-grid.md](https://github.com/uswds/uswds-site/blob/e10e348570a5ffe6540d78cf914041ff3a4c9966/_utilities/layout-grid.md#L511-L540)

**Comment by mejiaj** [comment]
@amyleadem can you confirm if this was addressed in your existing refactor work?

**Comment by amyleadem** [comment]
It looks like we haven't addressed this yet. We flagged it in the [code documentation audit](https://docs.google.com/spreadsheets/d/1YNid-FkaYHKJA7ezsJpCFhycACCpMPKmiNbo2QIf7lM/edit#gid=1473593582), but it has not yet been addressed. I've added a note in the spreadsheet that the item will be addressed in this issue.

**Comment by amyleadem** [comment]
Re-assigning to @mahoneycm to see if it has been addressed in #2276

**Comment by mahoneycm** [comment]
This issue should be completed as part of #2276 where we are [discussing](https://github.com/uswds/uswds-site/pull/2276#discussion_r1540237579) replacing this section with a link to the settings guidance or by capturing the relevant information in `site.data.settings.spacing.contents`

<img width="622" alt="image" src="https://github.com/uswds/uswds-site/assets/25211650/46ac374c-a7d1-4fc5-8083-a005a6e24332">
